src/libraries/System.CodeDom/Directory.Build.props

```diff
@@ -9,5 +9,7 @@ System.CodeDom.CodeObject
 System.CodeDom.Compiler.CodeDomProvider
 Microsoft.CSharp.CSharpCodeProvider
 Microsoft.VisualBasic.VBCodeProvider</PackageDescription>
+    <IncludePlatformAttributes>true</IncludePlatformAttributes>
+    <UnsupportedOSPlatforms>browser;ios;tvos;maccatalyst</UnsupportedOSPlatforms>
```

Comment:
@buyaa-n @joperezr @krwq @jeffhandley can you please double check if this is right? The platform analyzer warned for these OSs when I added the .NETCoreApp configuration to System.CodeDom.

Reply:
if this assembly only supported on certain platforms better to add a Supported attribute for those. If there is no such known list then it is OK, let's ask mono team review @marek-safar

Reply:
>  can you please double check if this is right? 

I think we don't have to support this on Android either

> if this assembly only supported on certain platforms better to add a Supported attribute for those.

I don't know what is supported set as the API is pretty weird and uses Process to launch unknown command line.

There is also a question about general single-file support here and the APIs could be considered as incompatible with singe-file too (/cc @agocke @vitek-karas)